src/BuildIntegration/Microsoft.NETCore.Native.Publish.targets

```diff
@@ -66,4 +66,10 @@
     
   </Target>
 
+  <Target Name="CopyNativePdb" Condition="'$(Configuration)'=='Debug'" AfterTargets="Publish">
```

Comment:
Why is this limited to '(Configuration)'=='Debug' ?

Reply:
Incorrect assumption on my part. ```dotnet publish``` seems to always produce pdbs, except when both <DebugSymbols> and <DebugType> are explicitly set. 
Would we want to honor them or copy the pdb regardless of property values?